notebooks/end-to-end-structured/solutions/5a_train_keras_vertex_babyweight.ipynb

```diff
@@ -0,0 +1,926 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #110.        for nc in NUMERICAL_COLUMNS:
</pre><p>nc -&gt; numerical_column</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/asl-ml-immersion/pull/206/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='21'/>

Reply:
<p>Done. Thanks.</p>